firmware/common/portapack_persistent_memory.cpp

```diff
@@ -89,26 +89,26 @@ enum data_structure_version_enum : uint32_t {
 
 static const uint32_t TOUCH_CALIBRATION_MAGIC = 0x074af82f;
 
-enum bits_t {
```

Comment:
@gullradriel you moved this out of the type in your debug PR? I put it back because it's meant to be internal.

Reply:
I was asking myself if it would not be better to have it global instead of having it repeat for each new added type but didn't go further